lib/msf/core/exploit/remote/http_client.rb

```diff
@@ -163,7 +166,9 @@ def connect(opts={})
       proxies,
       client_username,
       client_password,
-      comm: opts['comm']
+      comm: opts['comm'],
+      http_trace_proc_request: http_trace_proc_request,
+      http_trace_proc_response: http_trace_proc_response
```

Comment:
If we're passing in the subscriber logger object here, that would become something like:

```suggestion
      subscriber: http_subscriber_logger
```